src/servers/ZoneServer2016/handlers/commands/commands.ts

```diff
@@ -158,7 +160,7 @@ export const commands: Array<Command> = [
   },
   {
     name: "pop",
-    permissionLevel: PermissionLevels.DEFAULT,
+    permissionLevel: PermissionLevels.ADMIN,
```

Comment:
Isn't it a bit much? Like i guess you will remove every logs that display world pop? I'm ok for the main menu but that's all were not gonna do stuff like that to give the illusion there is players on the server

Reply:
A better idea could be do display other data too not only the actual player count but maybe the 24h connections ?

Reply:
> A better idea could be do display other data too not only the actual player count but maybe the 24h connections ?

http://charts.h1emu.com/Last%20month%20connections.html

Reply:
Even with the changes they can view the POP with Charts and the Website. Thats all thats needed. 

We are not removing it fully. Put in mind again daybreak had No charts, no pop command, no website display and servers didn't have the pop shown until 2018 as they never had that many players.

Players have told me they hated the POP count being shown too from day one, players would use it every 20mins to an hour. Im going on player feedback and I asked JSON to push this for me. :) 

Looking back at Jan 2023's update display was good without them.

Reply:
ok sounds good to me